### ISSUE TITLE
Default shortcut NVDA+shift+R conflicts with a native NVDA command in Excel

### ISSUE THREAD

**Issue by CyrilleB79** [sharkboyto/nao#8]
## Steps to reproduce
* Have Nao installed
* Open an Excel sheet
* Press NVDA+shift+R to define the current cell as row title.

## Expected result
The row title is defined.

## Actual result
Nao's error message is heard: "You must be in a Windows File Explorer window"

## Reasoning
1. Even if this shortcut is made modifiable in a future version of Nao, I think it is not a good practice to have a default add-on shortcut overriding NVDA's native shortcut, except if you enhance the native feature present in NVDA. Asking the user to have to change an add-on setting to get back a native NVDA feature is a pity.
2. It is still worse considering that Nao is of no use in Excel. Thus a useful feature is overridden by a command that reports an error message in any case (in Excel).
3. The shortcut NVDA+shift+R is used for Excel sheets and for Word tables. Thus, both shortcuts are impacted by the default Nao setting and both should be modified in case you wan to keep NVDA+shift+R for Nao.

## How to solve this issue
Here are various ideas to solve this issue:
* Assign another shortcut not present in NVDA
* Do not assign a shortcut at all and document the command as unassigned. And ask to the user in the documentation to assign in the input gesture dialog.
* Keep the same shortcut but restrict the command to places where it is useful (Explorer and other file managers); especially, do not bind this command in Word nor in Excel. I think that this solution is not very interesting however since the shortcut could be used elsewhere in the future by NVDA or an add-on to set row titles. For example set a row title in the Explorer file list.

## Context
Issue opened for myself and on behalf of other people on the French mailing list.
Especially, a user was asking on the mailing list why Excel's NVDA+shift+R was not working anymore. He had not realized that Nao was preempting this shortcut.
